### PR TITLE
add labels to mock client

### PR DESCRIPTION
**Pull request by maaland**

### CODE REVIEW

**Review by me-ydv-5 — DISMISSED** [review]
Thanks for adding this.

**Review by sanderland — CHANGES_REQUESTED** [review]
Could you have a look what else is missing, e.g. relationships?

**Comment by maaland** [comment]
@sanderland added what I could see was missing.

File: cognite/client/testing.py [hunk 1]
```diff
@@ -53,8 +59,10 @@ def __init__(self, *args, **kwargs):
         self.raw.databases = MagicMock(spec_set=RawDatabasesAPI)
         self.raw.tables = MagicMock(spec_set=RawTablesAPI)
         self.raw.rows = MagicMock(spec_set=RawRowsAPI)
+        self.relationships = MagicMock(spec_set=RelationshipsAPI)
         self.sequences = MagicMock(spec=SequencesAPI)
         self.sequences.data = MagicMock(spec_set=SequencesDataAPI)
+        self.synthetic_time_series = MagicMock(spec_set=SyntheticDatapointsAPI)
```
Comment (sanderland) [inline cognite/client/testing.py:65]: this is not a thing
https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_cognite_client.py

**Review by sanderland — CHANGES_REQUESTED** [review]

File: cognite/client/testing.py (hunk 1 above)
Comment (maaland) [reply cognite/client/testing.py:65]: Ah I see. I guess the same goes for `EntityMatching`

**Review by sanderland — APPROVED** [review]

**Pull request merged**